ui/ts/typings/chai/chai.d.ts

```diff
@@ -0,0 +1,308 @@
+// Type definitions for chai 2.0.0
```

Comment:
We have at least one type definition that is not from definitely typed (mithril), so we need to figure out how to mix-and-match; shouldn't be much of a problem.  Is it alright if I add a todo for this?